src/libtrx/include/libtrx/game/gun/const.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#define SHOTGUN_PELLET_SCATTER (DEG_1 * 20) // = 3640
+
+#define GUN_AMMO_CLIP 16
+#define GUN_AMMO_QTY (GUN_AMMO_CLIP * 2) // = 32
+
+#define SHOTGUN_SHELL_COUNT 2
+#define SHOTGUN_AMMO_CLIP 6
+#define SHOTGUN_AMMO_QTY (SHOTGUN_AMMO_CLIP * SHOTGUN_SHELL_COUNT) // = 12
+
+#if TR_VERSION == 1
+    #define MAGNUM_AMMO_CLIP 25
+    #define MAGNUM_AMMO_QTY (MAGNUM_AMMO_CLIP * 2) // = 50
+
+    #define UZI_AMMO_CLIP 50
+    #define UZI_AMMO_QTY (UZI_AMMO_CLIP * 2) // = 100
+#elif TR_VERSION == 2
+    #define MAGNUM_AMMO_CLIP 20
+    #define MAGNUM_AMMO_QTY (MAGNUM_AMMO_CLIP * 2) // = 40
+
+    #define UZI_AMMO_CLIP 40
+    #define UZI_AMMO_QTY (UZI_AMMO_CLIP * 2) // =  80
+
+    #define M16_AMMO_CLIP 40
+    #define M16_AMMO_QTY M16_AMMO_CLIP // = 40
+
+    #define HARPOON_AMMO_CLIP 3
+    #define HARPOON_AMMO_QTY HARPOON_AMMO_CLIP // = 3
+    #define HARPOON_BOLT_SPEED 150
+    #define HARPOON_RECOIL 4
+
+    #define GRENADE_AMMO_CLIP 1
+    #define GRENADE_AMMO_QTY (GRENADE_AMMO_CLIP * 2) // = 2
+    #define GRENADE_SPEED 200
+
+    #if defined(PSX_VERSION) && defined(JAPAN)
```

Comment:
We can forgo these defines as we never actually use them, and put a more meaningful `// TODO: ` entry.